corehq/apps/api/resources/v0_5.py

```diff
@@ -342,13 +342,38 @@ def obj_delete(self, bundle, **kwargs):
         return ImmediateHttpResponse(response=http.HttpAccepted())
 
 
-class WebUserResource(v0_1.WebUserResource):
-
-    class Meta(v0_1.WebUserResource.Meta):
-        detail_allowed_methods = ['get', 'put', 'delete']
-        list_allowed_methods = ['get', 'post']
+class WebUserResource(v0_1.UserResource):
+    role = fields.CharField()
+    is_admin = fields.BooleanField()
+    permissions = fields.DictField()
+
+    class Meta(v0_1.UserResource.Meta):
+        authentication = RequirePermissionAuthentication(Permissions.edit_web_users)
```

Comment:
I understand that this would be a feature change and might not be best to take it in this PR,
Should this now be `view_web_users` since we are exposing just `get` now?

Reply:
I think based on your other suggestion, I'm going to just undo this whole part of the change and leave it like it was before, with this line in `v0_1.WebUserResource`

Reply:
Sounds good.